calicoctl/commands/node/status.go

```diff
@@ -79,21 +79,44 @@ Description:
 				os.Exit(1)
 			}
 
-			// Connect to the bird socket and get the data if calico-node container is running
-			c, err := net.Dial("unix", "/var/run/calico/bird.ctl")
-			if err != nil {
-				log.Fatalf("Error connecting to the BIRD socket: %v", err)
+			versions := []string{"4", "6"}
+
+			// Loop over and try getting info about both IP versions
+			for _, v := range versions {
+				birdSuffix := ""
+				ipSep := "."
+				if v == "6" {
+					birdSuffix = "6"
+					ipSep = ":"
+				}
+				fmt.Printf("\nIPv%s BGP status", v)
+
+				// Connect to the bird socket and get the data
+				c, err := net.Dial("unix", fmt.Sprintf("/var/run/calico/bird%s.ctl", birdSuffix))
+				if err != nil {
+					log.Printf("Error connecting to BIRD socket: %v", err)
+					continue
+				}
+				defer c.Close()
+
+				fmt.Println()
+
+				_, _ = c.Write([]byte("show protocols\n"))
```

Comment:
I think the way BIRD console works is it always errors out even if the command succeeds:
```
0001 BIRD 1.5.0 ready.
**show protocols**
2002-name     proto    table    state  since       info
1002-static1  Static   master   up     20:47:33    
 kernel1  Kernel   master   up     20:47:33    
 device1  Device   master   up     20:47:33    
 direct1  Direct   master   up     20:47:33    
0000 
```

this still returns a non-nil error for some reason